react/features/conference/components/web/Conference.js

```diff
@@ -242,10 +235,6 @@ class Conference extends AbstractConference<Props, *> {
                         id = 'videospace'
                         onTouchStart = { this._onVidespaceTouchStart }>
                         <LargeVideo />
-                        {!_isParticipantsPaneVisible
-                         && <div id = 'notification-participant-list'>
-                             <KnockingParticipantList />
```

Comment:
Is this component used anywhere now?

Reply:
No. Just deleted.

Reply:
Or is used on native!? On native Conference.js I can see this import -> import { KnockingParticipantList } from '../../../lobby'; But I can see that on native is also an KnockingParticipantList component.

Reply:
We do need the native version for now, since the notifications are not that good there yet.